packages/xrpl/src/models/transactions/trustSet.ts

```diff
@@ -30,6 +30,10 @@ export enum TrustSetFlags {
   tfSetFreeze = 0x00100000,
   /** Unfreeze the trust line. */
   tfClearFreeze = 0x00200000,
+  /** Deep-Freeze the trust line -- disallow sending and recieving the said IssuedCurrency */
+  tfSetDeepFreeze = 0x00400000,
```

Comment:
would be convenient to add a comment along the lines of

`// Allowed only if the trustline is already regularly frozen, or if tfSetFreeze is set in the same transaction.`

Reply:
fixed in [c9ecd33](https://github.com/XRPLF/xrpl.js/pull/2873/commits/c9ecd33eb88848d25d1dca2a24c5270ac356cb04)